### DISCUSSION TITLE
Keine Verbindung zu Port 8082

### DISCUSSION THREAD

**Discussion by ape404** [buanet/ioBroker.docker#300] [Questions & Answers / Fragen & Antworten] [answered]
Habe ioBroker in einem Docker Container auf einem RPi 3B installiert. Das läuft soweit ok. Wenn ich aber die Adapter jarvis, web, oder cloud installiere wird keine Verbindung über den Port 8082 möglich. Es erscheint dort lediglich der Hinweis auf einen Verbindungfehler. Habe schon andere Ports probiert, gleiches Resultat. Das Log file sagt dazu:
<html><body>

jarvis.0 | 2022-11-13 15:28:11.887 | info | Terminated (ADAPTER_REQUESTED_TERMINATION): Without reason
-- | -- | -- | --
jarvis.0 | 2022-11-13 15:28:11.883 | info | terminating
jarvis.0 | 2022-11-13 15:28:11.879 | info | Adapter stopped und unloaded.
jarvis.0 | 2022-11-13 15:28:11.874 | info | Got terminate signal TERMINATE_YOURSELF

</body>
</html>

Was mache ich falsch?

**Comment by buanet** [comment]
Du musst schon dazu schreiben wie du deinen ioBroker Container betreibst. Entscheidend ist welche Netzwerkvariante du verwendest...

**Comment by ape404** [comment]
Ich habe ioBroker im Container mit dem scriptdocker run -p 8081:8081 --name iobroker -h iobroker buanet/iobrokergestartet. Weiter habe ich dann die Fragen zum Setup im Anmeledfenster beantwortet. Dort hat es aber soviel ich gesehen habe keine Fragen zum Netzwerk gehabt. Danke für Deine Hilfe. Albert

**Reply by buanet** [reply]
Mit deinem Docker run Befehl startest du den Container mit einem Bridge Netzwerk. Dabei musst du alle Ports die du verwenden möchtest durch reichen. Wenn du jetzt nicht weißt wovon ich spreche, such bitte mal nach "Docker Grundlagen", z. B. Bei Youtube. Weiterhin würde ich dir dringend raten deine iobroker Daten in ein Volume oder lokalen Dateipfad auszulagern.
Damit du deinen iobroker erfolgreich unter Docker betreiben kannst ist es unerlässlich dass du die Grundlagen beherrscht.

MfG,
André

**Comment by buanet** [comment, marked as answer]
Mit deinem Docker run Befehl startest du den Container mit einem Bridge Netzwerk. Dabei musst du alle Ports die du verwenden möchtest durch reichen. Wenn du jetzt nicht weißt wovon ich spreche, such bitte mal nach "Docker Grundlagen", z. B. Bei Youtube.

Weiterhin würde ich dir dringend raten deine iobroker Daten in ein Volume oder lokalen Dateipfad auszulagern.

Informationen, sowohl zum Netzwerk Thema, als auch zum Auslagern der ioBroker Daten findest du auch in der [Dokumentation zum Docker Image](https://docs.buanet.de/iobroker-docker-image/docs/).
Damit du deinen iobroker erfolgreich unter Docker betreiben kannst ist es unerlässlich dass du die Grundlagen beherrscht.

MfG,
André
